src/modules/real-time-metrics/reports/convert-beholder-to-chart.js

```diff
@@ -429,7 +441,11 @@ const formatBigNumbers = ({ report, data }) => {
   const aggregation = report.aggregationType
 
   const total = data[dataset].reduce((acc, current) => acc + current[aggregation || fieldName], 0)
-  const { unit, value } = formatDataUnit(total, report)
+  let { unit, value } = formatDataUnit(total, report)
+
+  if (report.id === '847143804009563421') {
+    unit = 'URLs'
+  }
```

Comment:
Sugiro crair uma const para esse valor
```suggestion
const TARGET_REPORT_ID_URLS = '847143804009563421';
  if (report.id === TARGET_REPORT_ID_URLS) {
    unit = 'URLs'
  }
```

Reply:
Só tem uma por enquanto. Caso haja mais, eu movo pra uma constante.